Zápis zo stretnutia merge (#7)
TIS2021-FMFI/prebalovanie: docs/zapisZoStretnutia.md

Kolega vám potom povie viacej o projekte a o tom čo sa teda od vás očakáva a ako by sme si to predstavovali, čo sa teda na tejto prevádzke robí atď., to by vám všetko povedal on, ale čo je za mňa, moje meno je Zuzana Andrášiková, písali sme si asi s vami, že?

Áno.

> Ja som v kontakte s vami teda kvôli tomu že mám na starosti spoluprácu so školami, či už strednými alebo vysokými, tam spadajú aj tieto projekty. Čiže ja som pre vás taká kontaktná osoba vzhľadom takých vecí organizačných, alebo keď budete potrebovať nejaké informácie, nie vyslovene k projektu, napríklad do školy alebo niečo podobné, alebo že by niečo nefungovalo, skrátka ako taký kordinátor. Tak kľudne sa ozvite na mňa, Prípadne ešte kolegyňa Zuzka Trízňová, ktorá, tiež myslímže, bola uvedená u profesora Petroviča ako ďalší kontakt. Čiže keby bolo čokoľvek, tak sme vám k dispozícii. Ale inak bude s vami komunikovať, čo sa projektu týka, takých tých technických vecí a špecifikácií, ohľadom toho projektu primárne teda kolega Mário Červenka alebo ešte jeden kolega Martin Stríž. Čiže oni dvaja sú takí garanti celého projektu a tej technickej stránky. Na Mária vám potom dám kontakt aby ste si mohli písať. V každom prípade budete mať potom nejakú mailovú komunikáciu, tak by som vás veľmi pekne chcela poprosiť aby ste tam udržiavali v kópii aj mňa alebo teda tú kolegyňu Trízňovú, vzhľadom na to že sa nám v minulosti stalo že potom nám unikali informácie že niečo ste od nás chceli aby sme pomohli niečo dohladať alebo niečo poskytnúť. Čo sa týka nejakej technickej podpory a napríklad [nezrozumiteľné] keď sa objednával a ďalej. Bolo to potom pre nás jednoduchšie keď sme si to vedeli vyhladať a kontaktovať jednu stranu, druhú stranu a informácie zisťovať, takže z tohto hladiska by sme vás chceli poprosiť, keď budete ohľadom čohokoľvek komunikovať, držte nás v kópii. A ja by som možno za mňa aj skončila a nechám teraz slovo kolegovi aby vám predstavil viacej o projekte na ktorom, chceme aby ste sa venovali.

Reviewer: ```suggestion
Ja som v kontakte s vami teda kvôli tomu že mám na starosti spoluprácu so školami, či už strednými alebo vysokými, tam spadajú aj tieto projekty. Čiže ja som pre vás taká kontaktná osoba vzhľadom takých vecí organizačných, alebo keď budete potrebovať nejaké informácie, nie vyslovene k projektu, napríklad do školy alebo niečo podobné, alebo že by niečo nefungovalo, skrátka ako taký kordinátor. Tak kľudne sa ozvite na mňa, Prípadne ešte kolegyňa Zuzka Trízňová, ktorá, tiež myslím že, bola uvedená u profesora Petroviča ako ďalší kontakt. Čiže keby bolo čokoľvek, tak sme vám k dispozícii. Ale inak bude s vami komunikovať, čo sa projektu týka, takých tých technických vecí a špecifikácií, ohľadom toho projektu primárne teda kolega Mário Červenka alebo ešte jeden kolega Martin Stríž. Čiže oni dvaja sú takí garanti celého projektu a tej technickej stránky. Na Mária vám potom dám kontakt aby ste si mohli písať. V každom prípade budete mať potom nejakú mailovú komunikáciu, tak by som vás veľmi pekne chcela poprosiť aby ste tam udržiavali v kópii aj mňa alebo teda tú kolegyňu Trízňovú, vzhľadom na to že sa nám v minulosti stalo že potom nám unikali informácie že niečo ste od nás chceli aby sme pomohli niečo dohladať alebo niečo poskytnúť. Čo sa týka nejakej technickej podpory a napríklad [nezrozumiteľné] keď sa objednával a ďalej. Bolo to potom pre nás jednoduchšie keď sme si to vedeli vyhladať a kontaktovať jednu stranu, druhú stranu a informácie zisťovať, takže z tohto hladiska by sme vás chceli poprosiť, keď budete ohľadom čohokoľvek komunikovať, držte nás v kópii. A ja by som možno za mňa aj skončila a nechám teraz slovo kolegovi aby vám predstavil viacej o projekte na ktorom, chceme aby ste sa venovali.
```

> Ok, tak volám sa Mário Červenka, pracujem tu ako projektový koordinátor. To znamená že nejaké prerábky skladu, nejaké optimalizácie a jednoducho všetko čo súvisí s operatívou a vedelo by nám to vlastne pomôcť, tak to ide väčšinou cezo mňa. Táto naša aktivita GEFCO VRP má na starosti príjem teraz nejakého materiálu od rôznych dodávateľov, následné zaskladnenie a potom expedícia. Pri tej expedícii je ešte prebal, a to bude vlastne taká tá vaša úloha. Všetko čo k nám chodí je väčšinou v kartóne. Ale dodávatelia, vačšinou sú to nejaké závody, buď je to trnavská automobilka alebo potom to ide niekam  do Francúzska, do [nezrozumiteľné], alebo nejaký iní závody. Takže vlastne vždycky ten materiálprebalujeme z kartónu do nejakých iných vratných obalov. Aby to jednoducho na tej linke operátori nejako lahšie zmanipulovali a jednoduchšie aby sa to vlastne vložilo, vložil, zakomponovali do tých áut. Takže ten prebal vlastne spočíva v tom, že na začiatku príde vlastne monitor. Monitor, to je operátor ktorý má na starosti nejakú skupinu ľudí.

Reviewer: ```suggestion
Ok, tak volám sa Mário Červenka, pracujem tu ako projektový koordinátor. To znamená že nejaké prerábky skladu, nejaké optimalizácie a jednoducho všetko čo súvisí s operatívou a vedelo by nám to vlastne pomôcť, tak to ide väčšinou cezo mňa. Táto naša aktivita GEFCO WRP má na starosti príjem teraz nejakého materiálu od rôznych dodávateľov, následné zaskladnenie a potom expedícia. Pri tej expedícii je ešte prebal, a to bude vlastne taká tá vaša úloha. Všetko čo k nám chodí je väčšinou v kartóne. Ale dodávatelia, vačšinou sú to nejaké závody, buď je to trnavská automobilka alebo potom to ide niekam  do Francúzska, do [nezrozumiteľné], alebo nejaké iné závody. Takže vlastne vždycky ten materiál prebalujeme z kartónu do nejakých iných vratných obalov. Aby to jednoducho na tej linke operátori nejako lahšie zmanipulovali a jednoduchšie aby sa to vlastne vložilo, vložil, zakomponovali do tých áut. Takže ten prebal vlastne spočíva v tom, že na začiatku príde vlastne monitor. Monitor, to je operátor ktorý má na starosti nejakú skupinu ľudí.
```

Ukážka odhadovaného návrhu.-----------
Hlavička a prihlasovacie údaje (kontrola prihlásenia „Zle zadané meno alebo heslo“ alebo niečo podobné..)

Meno heslo – nejakých 8znakov (číslice, písmená, špeciálne znaky)

-Hlavné menu – to by bolo rozdelené na referencia, prebal ( funkcia samostatného prebaľovania), operatíva( história záznamov, export údajov ktoré sú zahrnuté v aplikácii, prístupy, skupiny, práva), profil (meno a priezvisko), a nastavenia ( meno, login, kto sa kedy prihlásil, čo tam spravil, na aký PC/IP adresu sa to spustilo ),

-Nastavenia SKU- nejaká tabuľka zo všetkými údajmi, ktoré bude potrebovať ( referencia SKU, COFOR, destinácia, ....). Vloženie cez nejaký editor, aby to bolo čo najjednoduchšie pre operátora. Fotky aby sa tam zmestili (na A4) – nastaviť veľkosť/rozmer, ostatok sú číselné hodnoty, poznámka – maximálny počet znakov nastaviť.

-Spustenie prebaľovania- (to je na nás, to si nejako nevie pán predstaviť), ak sa prebaľuje, tak jedna referencia. 
Operátor musí mať svoj čiarový kód, aby vedeli identifikovať, ktorý operátor čo robil, aby ich vedeli rozoznať/rozdeliť. Musí sa prihlásiť aj operátor (nejaký bar/čiarový kód/ 4 písmená).

> Niekedy 1referenciu prebaľuje 1operátor, keď je ťažšia tak ju môžu prebaľovať 2operátori, alebo keď sa blíži koniec-začiatok (predel zmien)  zmeny, tak si vypomôžu operátori s prácou. Mal by sa vedieť prihlásiť aj druhý operátor. Môže byť funkcia na pridanie ďalšieho operátora.

Reviewer: ```suggestion
Niekedy 1 referenciu prebaľuje 1 operátor, keď je ťažšia tak ju môžu prebaľovať 2 operátori, alebo keď sa blíži koniec-začiatok (predel zmien)  zmeny, tak si vypomôžu operátori s prácou. Mal by sa vedieť prihlásiť aj druhý operátor. Môže byť funkcia na pridanie ďalšieho operátora.
```

-Používatelia – chceli by aplikáciu rozdeliť aby operátor videl len nejakú časť a len nejakú funkcionalitu alebo tlačidlo prebal. Aby nevidel ostatné veci, aby tam nevedel dávať údaje alebo ich mazať. Čiže rozlíšiť používateľov.

-Skupiny – vytvoríš čo chceš aby ten používateľ videl poprípade vedel čo má kde exportovať, čítať. Hore je okno kde vyberieš skupinu a vyberieš možnosti práv ktoré chceš aby mala.
Profil – taký úplný základ, aby vedeli kto si, kedy si bol vytvorený(účet), ID .

Udalosti – login aby sme vedeli kto čo spravil, poprípade kde nastala chyba – padla aplikácia, aby sa vedelo kto čo/to spravil, čo sa v systéme udialo.

Odhlásenie – hore v rohu je tlačidlo Odhlásiť sa, a hneď potom vyskočí začiatočné užívateľské okno.
Menu užívateľa, v logu bude GEFCO, môže byť aj naša značka, prezentácia – vizuál je podľa nich ale môžeme ju/ho aj zmeniť, hlavne aby to fungovalo.

Prihlásenie- chcú využívať scanery na prihlásenie(čiarový kód), čiarový kód sa premietne ako ID číslo.

Scanery - bežný /obyčajný

Referencia má prefix, aj IDP má prefix (to nám pošle)

tri údaje sa budú scanovať.

> Etikety ´neriešime, to si tam operátor nalepí/dodá.

Reviewer: ```suggestion
Etikety netreba riešiť, to si tam operátor nalepí/dodá.
```

Cieľ pre GEFCO je rýchlejšia tvorba štandardov a následné využitie pri pracovnom postupe/zjednodušenie pracovných postupov.

Na príjme by prišla nová paleta, bola by naskenovaná a keby neexistoval štandard/referencia v databáze, prišiel by zamestnanec a vytvoril štandard.

Čas prebalu je na jednu referenciu/box.

Budú robiť na úrovni boxov.

Fotky sa odfotia a nahrajú do počítača kde sa pridajú do databázy.

Zistenie typu skeneru.

Importovanie a exportovanie dát má byť možné.

Zatiaľ GEFCO funguje v exceli.

> Prezentácia by sa dala poslať týmu.

Reviewer: ```suggestion
Prezentácia by sa dala poslať tímu.
```

Gefco má prenajatý server.

> Dokument s korporátnou identitou sa pošle týmu.

Reviewer: ```suggestion
Dokument s korporátnou identitou sa pošle tímu.
```
